### PR TITLE
Switch postgres client lib

### PR DESCRIPTION
**Pull request by edongashi**

### CODE REVIEW

File: src/AdminTab/TableList.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ export const TableList: FunctionComponent = () => {
 
   return (
     <div className="TableList">
-      <Table dataSource={tableList} loading={tableListIsLoading} rowKey="name">
+      <Table dataSource={tableList} loading={tableListIsLoading} rowKey="key">
```
Comment (pdobacz) [inline src/AdminTab/TableList.tsx:32]: why this change?

File: src/main/tables.ts [hunk 2]
```diff
@@ -32,45 +25,31 @@ async function syncTables(): Promise<void> {
...
-  const client = new pg.Client(connectionConfig);
-  await client.connect();
-
-  try {
-    await client.query(`DROP TABLE public."${tableName}";`);
-    await syncTables();
-  } finally {
-    client.end();
-  }
+  await sql`DROP TABLE public.${sql(tableName)}`;
```
Comment (pdobacz) [inline src/main/tables.ts:51]: I presume the `sql` handles correct quoting?

File: src/main/tables.ts [hunk 3]
```diff
@@ -170,41 +157,41 @@ export async function importCSV(
...
-    await client.query(`DROP TABLE IF EXISTS "${tableName}"`);
-    await client.query(`CREATE TABLE "${tableName}" (${columnsSQL})`);
-    await copyRowsFromCsvFile(client, signal, fileName, fieldProcessors, tableName);
-    if (aidColumns.length > 0) {
-      if (aidColumns.includes(ROW_INDEX_COLUMN)) {
-        await client.query(`ALTER TABLE "${tableName}" ADD COLUMN IF NOT EXISTS "${ROW_INDEX_COLUMN}" SERIAL`);
+    await sql.begin(async (sql) => {
+      await sql`DROP TABLE IF EXISTS ${sql(tableName)}`;
+
+      const columnsSQL: SqlFragment[] = columns.map((column, i) =>
```
Comment (pdobacz) [inline src/main/tables.ts:164]: does this make sql-injection (as in the comment) fully protected against? if so, you can mark the #13 as fixed by this PR

File: src/main/tables.ts [hunk 4]
```diff
@@ -170,41 +157,41 @@ export async function importCSV(
...
+        await sql`CALL diffix.mark_personal('${sql(tableName)}', ${aidColumnsSQL})`;
+      } else {
+        await sql`CALL diffix.mark_public('${sql(tableName)}')`;
       }
-      const aidColumnsSQL = aidColumns.map((aidColumn) => `'"${aidColumn}"'`).join(', ');
-      await client.query(`CALL diffix.mark_personal('"${tableName}"', ${aidColumnsSQL});`);
-    } else {
-      await client.query(`CALL diffix.mark_public('"${tableName}"');`);
-    }
-    await client.query(`GRANT SELECT ON "${tableName}" TO "${postgresConfig.trustedUser}"`);
```
Comment (pdobacz) [inline src/main/tables.ts:196]: is removing grant select intended here? it's not necessary because we give them SUPERUSER, but I think it's safer to not rely on that and grant.

**Review by pdobacz — DISMISSED** [review]

File: src/main/tables.ts (hunk 2 above)
Comment (edongashi) [reply src/main/tables.ts:51]: Yes, `${someString}` adds single quotes, and `${sql(someString)}` adds double quotes. A bit too much magic and takes getting used to.

File: src/main/tables.ts (hunk 3 above)
Comment (edongashi) [reply src/main/tables.ts:164]: Yes, no more injections, except the `Hack:` part which is a workaround...

File: src/main/tables.ts (hunk 4 above)
Comment (edongashi) [reply src/main/tables.ts:196]: Accident. Thanks for the catch.

File: src/AdminTab/TableList.tsx (hunk 1 above)
Comment (edongashi) [reply src/AdminTab/TableList.tsx:32]: I'm confused why tables have both `key` and `name`? Given the presence of a `key` prop, I assume that will be our primary identifier. Should we drop it?

Comment (pdobacz) [reply src/AdminTab/TableList.tsx:32]: tbh, I didn't dig deep. I recognize this `rowKey` addition from a recent PR of @cristianberneanu so I thought I'd make sure this change is not a leftover of some sort.

Now I looked, and indeed we have a `key` field, possibly specifically for this and I guess if `rowKey=name` does what I think it does then we might drop it instead.

**Review by pdobacz — APPROVED** [review]
